lib/robots/dor_repo/accession/reset_workspace.rb

```diff
@@ -11,7 +11,8 @@ def initialize
         end
 
         def perform_work
-          object_client.workspace.reset
+          object_client.workspace.reset(workflow: 'accessionWF', lane_id:)
```

Comment:
so this is calling an async job to be run??  Maybe a comment could help

Reply:
Comment added.